cnf-certification-test/platform/bootparams/bootparams.go

```diff
@@ -94,12 +94,12 @@ func getGrubKernelArgs(env *provider.TestEnvironment, nodeName string) (aMap map
 	return arrayhelper.ArgListToMap(grubSplitKernelConfig), nil
 }
 
-func getCurrentKernelCmdlineArgs(cut *provider.Container) (aMap map[string]string, err error) {
+func getCurrentKernelCmdlineArgs(env *provider.TestEnvironment, nodeName string) (aMap map[string]string, err error) {
 	o := clientsholder.GetClientsHolder()
-	ctx := clientsholder.NewContext(cut.Namespace, cut.Podname, cut.Name)
+	ctx := clientsholder.NewContext(env.DebugPods[nodeName].Namespace, env.DebugPods[nodeName].Name, env.DebugPods[nodeName].Spec.Containers[0].Name)
 	currentKernelCmdlineArgs, errStr, err := o.ExecCommandContainer(ctx, kernelArgscommand)
```

Comment:
why not use "outStr, errStr, err := crclient.ExecCommandContainerNSEnter(getListeningPortsCmd, cut)" instead. We would keep the same logic but not rely on commands installed on the container

Reply:
Yes this make sense, I will switch.

Reply:
@edcdavid I don't think we need `nsenter` here, as we're cat'ing the cmdline file from each OCP node's OS using the "cat" command that (we know) should exist in our debug pods (UBI-based images). I believe we should also add the `/host` prefix to the `kernelArgsCommand`:
```
	kernelArgscommand     = "cat /host/proc/cmdline"
```